src/Dialog/Dialog.js

```diff
@@ -74,6 +70,10 @@ export default class Dialog extends Component {
      */
     onEntering: PropTypes.func,
     /**
+     * Callback fires when the escape key is pressed and the modal is in focus
+     */
+    onEscapeKeyUp: PropTypes.func,
```

Comment:
This should be below the next one--which has a `disable-line` comment so that the `onEnterxxx` props can be listed in order of their execution.